README.md

```diff
@@ -66,6 +66,7 @@ Attempts to load all TOML files to verify syntax.
 
 #### `check-vcs-permalinks`
 Ensures that links to vcs websites are permalinks.
+  - `--github-enterprise-domain DOMAIN` - Add check for specified GitHub Enterprise domain.
```

Comment:
maybe `--additional-domain` is simpler? it doesn't necessarily have to be github enterprise, just something with similar url structure

Reply:
Applied it!